scripts/CEE/worker-node-operations/README.md

```diff
@@ -0,0 +1,48 @@
+# Worker Node Operations (Cordon/Uncordon / Drain / Reboot)
+
+## Description
+
+This script is responsible for performing Cordoning/Uncordoning, Draining and Rebooting operations on Worker nodes, giving the options to perform such operations as required.
+
+## Available Options
+
+The managed script provides the following actions:
+
+- cordon: To cordon worker nodes
+- uncordon: To uncordon worker nodes
+- drain: This will cordon and then drain worker nodes
+- reboot: This will cordon, drain and then reboot worker nodes
+
+## Usage
+
+**ATTENTION** ⚠️ This script must only be used by the MCS (Managed Cloud Services) CRU (Crew Response Unit) team.
+- If you need this script to be used, contact a MCS CRU member.
```

Comment:
```suggestion
- If you need this script to be used, contact a MCS CRU member.
```
Why not SRE as well?

Reply:
This is only a recommendation for the MCS members (who is not part of the CRU) to have a CRU member supervision.